package.json

```diff
@@ -74,5 +75,8 @@
     "angular": "1.x",
     "angular-simple-logger": "0.1.x",
     "leaflet": "0.7.x"
+  },
+  "engines": {
+    "npm": ">=3.0.0"
```

Comment:
Actually can we not have this restriction. I am still on npm 2 and npm 3 has been nothing but slow for me.